packages/css/src/common/common.client.scss

```diff
@@ -20,7 +19,55 @@ $color-btn-error-bg: #fff3f5 !default;
 $color-btn-tertiary-bg: #eef !default;
 $color-btn-tertiary-bg-darker: #e4e4ff !default;
 $color-btn-tertiary-bg-lighter: #f8f8ff !default;
+$color-oceanBlue: #4976ba !default;
 
 $component-border-radius: 8px !default;
 
 $transition-duration: 0.2s !default;
+
+$sizing-u0: 0;
```

Comment:
d'où viennent ces constantes ? c'est généré depuis le figma ?
Les noms ne sont pas très parlants et pas sûr qu'ils soient tous utiles

Reply:
@samuel-gomez 
ca vient du dune layout, copie colle, mais ouais je suis d'accord que c pas top.
que proposes de faire ?
pour chaque component mettre tout ce qui le concerne en dur directement sans variable ?

Reply:
Ici, on repart de zéro avec le Look And Feel, je serais d'avis de créer de nouvelles constantes à chaque fois qu'on en a le besoin.
Pour ce qui est des noms de variables, il faudrait s'accorder avec les UX.
@astrinxit66 qu'en penses-tu ?

Reply:
Actuellement je ne vois pas de `sizing` dans les fondations du L&F. Quant aux `spacings`, voici ce qui est spécifié :
<img width="749" alt="image" src="https://github.com/AxaFrance/design-system/assets/15631355/0e4ef12c-7e6e-445f-a4f8-cc4a5a310d13">

Reply:
@jmraxa